tests/integration/commands/declare_command_test.py

```diff
@@ -26,6 +34,67 @@ def compiled_contract_path_fixture() -> Path:
     return Path("./build/main.json")
 
 
+async def test_declaring_cairo1_contract(
```

Comment:
We actually should a test in which the compiled class hash doesn't match the actual one, and check what happens there. Can be done in a separate issue, but in this release.

Reply:
The behavior from user's perspective is important, i think it's a good case for e2e test. UX is something we really want to have done right on declare, since it's a pain point in 0.11

Reply:
Okay, I'll introduce the test in this PR since it's related.

Reply:
Actually it doesn't make sense to test this. `compiled_class_hash` was a leftover argument in the tests. It is not being accepted by the command itself.